packages/wayland/weston/config/weston.ini

```diff
@@ -0,0 +1,13 @@
+[core]
+idle-time=0
+
```

Comment:
Rockchip needs `repaint-window=15` in `[core]` to be able to render at 60 fps, would that hurt other platforms?